sellercenter-setting/src/main/java/com/gfg/sellercenter/setting/entity/Setting.java

```diff
@@ -0,0 +1,20 @@
+package com.gfg.sellercenter.setting.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+@EqualsAndHashCode
+public class Setting {
+
+    @NotNull
+    private int sellerId;
+
+    @NotNull
+    private String label;
+
+    @NotNull
```

Comment:
Value can be null, at least it is nullable field in database (don't know, maybe somewhere on saving we do have validation and there are no nulls)

Reply:
Ticket created https://gfgroup.atlassian.net/browse/TML-32438

Reply:
> Ticket created https://gfgroup.atlassian.net/browse/TML-32438

:+1: